lib/runtime.cpp

```diff
@@ -335,6 +335,13 @@ RuntimeSettings::RuntimeSettings(const std::string& prefix_, const std::string&
     } else {
         validate_schema = defaults::VALIDATE_SCHEMA;
     }
+
+    const auto settings_validate_schema_on_startup_it = settings.find("validate_schema_on_startup");
+    if (settings_validate_schema_on_startup_it != settings.end()) {
+        validate_schema_on_startup = settings_validate_schema_on_startup_it->get<bool>();
+    } else {
+        validate_schema_on_startup = defaults::VALIDATE_SCHEMA_ON_STARTUP;
+    }
```

Comment:
I know the other code isn't doing it like that, but definitely, writing
`validate_schema_on_startup = settings.value("validate_schema_on_startup", defaults::VALIDATE_SCHEMA_ON_STARTUP);`
should be the correct choice here

Reply:
Changed, this was really just done for consistency with the rest of the code.

Reply:
removed